sql/ts/tests/tests.ts

```diff
@@ -1320,6 +1320,56 @@ const watchChangesTests: (asWorker: boolean) => Test[] = (
   ];
 };
 
+const dateTests = (asWorker: boolean) => {
+  let dn = (name: string, asWorker: boolean) => {
+    return n("Date: " + name, asWorker);
+  };
+  return [
+    {
+      name: dn("Add days", asWorker),
+      fun: async (skdb: SKDB) => {
+        return await skdb.exec(
```

Comment:
Could use `.scalarValue()` on the result of this `exec` and then just expect the value "2023-01-03" (and similarly throughout) to avoid hardcoding the `"col<0>"` and simplify things a bit